src/chat.rs

```diff
@@ -4226,4 +4230,34 @@ mod tests {
         assert!(msg.is_forwarded());
         Ok(())
     }
+
+    #[async_std::test]
+    async fn test_can_send_group() -> Result<()> {
+        let alice = TestContext::new_alice().await;
+        let bob = Contact::create(&alice, "", "bob@f.br").await.unwrap();
```

Comment:
As the test returns `Result`, you can use `?` instead of `unwrap()` for `Result`s.